src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -539,6 +730,43 @@ std::string Visitor::visitTypesafe(Parser::DataBlockValueContext* ctx) {
   AD_FAIL()  // Should be unreachable.
 }
 
+// ____________________________________________________________________________________
+GraphPatternOperation::Minus Visitor::visitTypesafe(
+    Parser::MinusGraphPatternContext* ctx) {
+  return {visitTypesafe(ctx->groupGraphPattern())};
+}
+
+// ____________________________________________________________________________________
+using GraphPattern = ParsedQuery::GraphPattern;
+namespace {
+GraphPattern wrap(GraphPatternOperation op) {
+  auto pattern = GraphPattern();
+  pattern._children.emplace_back(std::move(op));
+  return pattern;
+}
```

Comment:
Wrappers like these really give you a hint, that the type is not where it should be:)